src/widgets/about-school/ui/about.scss

```diff
@@ -14,32 +14,22 @@
         }
       }
 
-      .info {
-        display: flex;
-        flex-direction: column;
-        gap: 16px;
-        min-width: 300px;
-
-        > .paragraph {
-          @include media-laptop {
-            font-size: 18px;
-            line-height: 28px;
-          }
-
-          @include media-tablet {
-            font-size: 16px;
-            line-height: 24px;
-          }
+      .about-school-paragraph {
```

Comment:
We are overriding our `Paragraph` styles here, I don't think that's right. Maybe we should make several `font-size` options if necessary. Or, if it's only in one place on the site, remove the style override.

Reply:
This is a specific behavior, it only occurs here and if it is replaced with the standard behavior, the current implementation will differ from the layout...

Reply:
The same specific behavior in study-path-paragraph.

Reply:
I don't see any reason why it should be different in this block. I suggest making it the same everywhere.

Reply:
There are differences in font sizes on media-laptop.

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/c2a047a0dedc9dbacb91d4e2a6adb6117bb32111).